.husky/pre-commit

```diff
@@ -3,3 +3,14 @@
 
 yarn lint-staged
 yarn type:check
+
+# this automatically builds plugin js files if you forgor
+! git diff HEAD --name-only | grep plugin
+if [ $? -eq 1 ]; then
+  yarn plugin
+  ! git status -u | grep -E 'plugin/build/plugin.js|plugin/build/plugin.js.map'
```

Comment:
1. Why not `-e`?
2. `.` will match any character, we should either escape the dots or match just `plugin/build` (this also resolves 1.).

Reply:
actually -e is not needed anyway and I actually forgot escaping